frontends/qt/server/server.go

```diff
@@ -139,6 +146,17 @@ func serve(
 			SystemOpenFunc:      system.Open,
 			UsingMobileDataFunc: func() bool { return false },
 			NativeLocaleFunc:    func() string { return nativeLocale },
+			GetSaveFilenameFunc: func(suggestedFilename string) string {
+				cSuggestedFilename := C.CString(suggestedFilename)
+				cFilename := C.getSaveFilename(getSaveFilenameFn, cSuggestedFilename)
+				if cFilename == nil {
+					return ""
```

Comment:
isn't this a little memory leak because `cSuggestedFilename` isn't freed here? defer C.free?

Reply:
good catch